include/swift/Reflection/ReflectionContext.h

```diff
@@ -31,6 +35,31 @@
 #include <vector>
 #include <unordered_map>
 
+#ifdef __APPLE__
+#ifndef __LP64__
```

Comment:
I think that `__POINTER_WIDTH__` would be nicer and is used elsewhere in the codebase.

Reply:
I used `__LP64__` because that's what `macho/getsect.h` uses to choose which `struct` to use. So I think it's best here, even though I agree `__POINTER_WIDTH__` would be better in general.